.github/workflows/make-version.yml

```diff
@@ -0,0 +1,49 @@
+name: Make Version
+
+on:
+  workflow_dispatch: { }
+
+
+env:
+  RELEASE_COMMIT: ${{ github.sha }}
+
+jobs:
+  bump-version:
+    permissions:
+      id-token: write
+      contents: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    outputs:
+      RELEASE_VERSION: ${{ steps.set-release-version.outputs.RELEASE_VERSION }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11  # v4.1.1
+        with:
+          ref: ${{ github.ref }}
+      - name: Setup NodeJS
+        uses: actions/setup-node@b39b52d1213e96004bfcb1c61a8a6fa8ab84f3e8 # v3.6.0
+        with:
+          node-version: "20"
+          cache: "npm"
+      - name: Setup auth tokens
```

Comment:
Before this step we need this:

```yml
- name: Setup dependencies
   uses: ./.github/actions/cached-node-modules
```

otherwise it will get `lerna` from npm, which can be a problem.

(In the v2 branch we are also [adding a flag](https://github.com/aws-powertools/powertools-lambda-typescript/blob/fc59672e2bf87cd565be6880c694fc83b102e055/.github/actions/cached-node-modules/action.yml#L7-L9) to that custom action to skip building the packages, which is not needed in this context, so it'll be faster)

Reply:
![image](https://github.com/aws-powertools/powertools-lambda-typescript/assets/7353869/1a27dae4-80b7-484d-9861-2b88c68a0aef)

Reply:
Do you mean after this step similar to what we have in `make-release.yml`?

Reply:
Yep, you can replace the setup node with this one, like in the other workflows

Reply:
done

Reply:
forgot to commit?

Reply:
not it's right below, no?